### PR TITLE
Removes felinid language

### PR DESCRIPTION
**Pull request by ReddicusDragon**
# Document the changes in your pull request
Xantam once said that donor features should not give a material benefit for donors. Felinids having a meta language seems like a material benefit.
# Wiki Documentation
English bit for tcomms is no longer 2048 and is instead 1024
Felinid language is gone
# Changelog
:cl:  
rscdel: Felinid language has been forcibly erased from every nook and cranny of history, including native speakers' brains.
/:cl:

### CODE REVIEW

**Comment by JamieD1** [comment]
Madlad did it.

**Comment by SomeguyManperson** [comment]
🐴

**Comment by Skrem7** [comment]
Lore friendly

File: code/modules/language/language_holder.dm [hunk 1]
```diff
@@ -363,14 +363,11 @@ Key procs
 							/datum/language/ratvar = list(LANGUAGE_ATOM),)
 
 /datum/language_holder/felinid
-	understood_languages = list(/datum/language/common = list(LANGUAGE_ATOM),
-								/datum/language/felinid = list(LANGUAGE_ATOM))
-	spoken_languages = list(/datum/language/common = list(LANGUAGE_ATOM),
-							/datum/language/felinid = list(LANGUAGE_ATOM))
+	understood_languages = list(/datum/language/common = list(LANGUAGE_ATOM))
```
Comment (SomeguyManperson) [inline code/modules/language/language_holder.dm]: add english or give them the human language holder

**Comment by MajManatee** [comment]
legend

**Comment by Bibby0110** [comment]
good pr

**Comment by JamieD1** [comment]
JamieSeal

**Review by MajManatee — COMMENTED** [review]
You could also remove the icon that went next to the text when they talk too.

**Comment by MajManatee** [comment]
ling cannot face the truth

**Comment by ThatLing** [comment]
@MajManatee Stop using it on PRs you personally agree with, this isn't objectively the best PR

**Comment by Hopekz** [comment]
> @MajManatee Stop using it on PRs you personally agree with, this isn't objectively the best PR

Meme maintainer uses meme tags in a meme way

**Review by cuackles — APPROVED** [review]

**Review by Moltijoe — APPROVED** [review]
based

**Comment by patpol4** [comment]
Very good

**Comment by Sniblet** [comment]
*me says (in felinid so humans can't understand or its failRP), "They can't silence us"

**Pull request merged**